src/js/common/schemaform/FormApp.jsx

```diff
@@ -1,221 +1,55 @@
 import React from 'react';
-import moment from 'moment';
-import Scroll from 'react-scroll';
-import { withRouter } from 'react-router';
 import { connect } from 'react-redux';
 
 import FormNav from './FormNav';
 import FormTitle from './FormTitle';
 import AskVAQuestions from './AskVAQuestions';
-import {
-  LOAD_STATUSES,
-  PREFILL_STATUSES,
-  SAVE_STATUSES,
-  setFetchFormStatus,
-  fetchInProgressForm
-} from './save-load-actions';
-import LoadingIndicator from '../components/LoadingIndicator';
-
-import { isInProgress } from '../utils/helpers';
-
-const Element = Scroll.Element;
-const scroller = Scroll.scroller;
-const scrollToTop = () => {
-  scroller.scrollTo('topScrollElement', window.VetsGov.scroll || {
-    duration: 500,
-    delay: 0,
-    smooth: true
-  });
-};
-
-moment.updateLocale('en', {
-  meridiem: (hour) => {
-    if (hour < 12) {
-      return 'a.m.';
-    }
-    return 'p.m.';
-  },
-  monthsShort: [
-    'Jan.',
-    'Feb.',
-    'Mar.',
-    'Apr.',
-    'May',
-    'June',
-    'July',
-    'Aug.',
-    'Sept.',
-    'Oct.',
-    'Nov.',
-    'Dec.'
-  ]
-});
+import RoutedSavableApp from './RoutedSavableApp';
 
 /*
  * Primary component for a schema generated form app.
  */
 class FormApp extends React.Component {
-  componentWillMount() {
-    window.addEventListener('beforeunload', this.onbeforeunload);
-    if (window.History) {
```

Comment:
@jbalboni question about this though: I removed all scrolling functionality from `FormApp`, that all lives in `RoutedSavableApp`. Does this code still need to be in `FormApp` then?

Reply:
It affects the scrolling that's done in FormPage and other places, so I think so.

I'm thinking that I may end up pulling the scrolling functionality in FormPage into here at some point, too. It has a couple issues because it's not at the top level.

Reply:
gotcha, okay will add this back in